src/hooks/useFeaturesSort.ts

```diff
@@ -175,7 +175,7 @@ const compareNullableDates = (
     a: Date | null | undefined,
     b: Date | null | undefined
 ): number => {
-    return a && b ? a.getTime() - b.getTime() : a ? 1 : b ? -1 : 0;
+    return a && b ? a?.getTime?.() - b?.getTime?.() : a ? 1 : b ? -1 : 0;
```

Comment:
Seems like the previous version should work.

Reply:
I had some kind of edge case, where it didn't and broke the UI, similar to the issue with 4.10 bug. It's a temporary fix anyway, since we will migrate this to new table setup this week.